app/helpers/ExerciseConfig/Pipeline/Box/Boxes/JudgeBox.php

```diff
@@ -29,11 +29,7 @@ class JudgeBox extends Box
   public static $EXPECTED_OUTPUT_PORT_KEY = "expected-output";
   public static $DEFAULT_NAME = "ReCodEx Judge";
 
-  /* TYPES OF JUDGES */
-  public static $RECODEX_NORMAL_TYPE = "recodex-judge-normal";
-  public static $RECODEX_SHUFFLE_TYPE = "recodex-judge-shuffle";
-  public static $DIFF_TYPE = "diff";
-  public static $DIFF_BINARY = "/usr/bin/diff";
+  /** Translation of judge type to command and args. The first item is the default. */
```

Comment:
Why is this doc comment here?

Reply:
Because the author of this PR is an idiot, who cannot copy-paste stuff properly.